enclave_manager/avalon_enclave_manager/graphene_wpe/graphene_wpe_manager.py

```diff
@@ -242,9 +266,7 @@ def main(args=None):
 
     try:
         EnclaveManager.parse_command_line(config, remainder)
-        logger.info("Initialize GrapheneWPEManager")
```

Comment:
Why removed the comments? Anything specific.

Reply:
It was accidental. Reverted the log.